x-pack/plugins/threat_intelligence/cypress/e2e/indicators.cy.ts

```diff
@@ -46,23 +46,26 @@ import { login } from '../tasks/login';
 import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
 import { selectRange } from '../tasks/select_range';
 
-before(() => {
-  login();
-});
-
 const THREAT_INTELLIGENCE = '/app/security/threat_intelligence/indicators';
 
 const URL_WITH_CONTRADICTORY_FILTERS =
   '/app/security/threat_intelligence/indicators?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!((%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:file),type:phrase),query:(match_phrase:(threat.indicator.type:file))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:url),type:phrase),query:(match_phrase:(threat.indicator.type:url)))),timeRange:(from:now/d,to:now/d))';
 
 describe('Invalid Indicators', () => {
+  before(() => {
+    login();
```

Comment:
@patrykkopycinski same comment for this file, I feel like we should have the
```
before(() => {
  login();
});
```
in the second `describe` section as well no?